internal/woc_client/woc_client.go

```diff
@@ -276,3 +286,61 @@ func (w *WocClient) TopUp(ctx context.Context, mainnet bool, address string) err
 
 	return nil
 }
+
+func (w *WocClient) GetRawTxs(ctx context.Context, mainnet bool, ids []string) ([]*wocRawTx, error) {
+	const maxIDsNum = 20 // value from doc
+
+	var result []*wocRawTx
+
+	net := "test"
+	if mainnet {
+		net = "main"
+	}
+
+	for len(ids) > 0 {
+		bsize := min(maxIDsNum, len(ids))
+		batch := ids[:bsize]
+
+		txs, err := w.getRawTxs(ctx, net, batch)
+		if err != nil {
+			return nil, err
+		}
+
+		result = append(result, txs...)
+		ids = ids[bsize:] // move batch window
+	}
+
+	return result, nil
+}
+
+func (w *WocClient) getRawTxs(ctx context.Context, net string, batch []string) ([]*wocRawTx, error) {
+	payload := map[string][]string{"txs": batch}
+	body, _ := json.Marshal(payload)
```

Comment:
There is an unhandled error

Reply:
Please correct me if I'm wrong, but `json.Marshal` would return an error only if the passed object is invalid JSON (e.g., circular references, invalid string, etc.). Here, the JSON data is created by me (the developer), and it's a one-key map. In my opinion, checking for an error here is unnecessary - WDYT?